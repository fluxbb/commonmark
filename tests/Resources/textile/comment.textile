This is a paragraph

###. This is a comment

This is a paragraph