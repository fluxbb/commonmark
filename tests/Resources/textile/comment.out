<p>This is a paragraph</p>

<p>This is a paragraph</p>